lib/protocol/networks.js

```diff
@@ -84,7 +84,23 @@ main.brontidePort = 44806;
 
 main.checkpointMap = {
   1008: Buffer.from(
-    '0000000000001013c28fa079b545fb805f04c496687799b98e35e83cbbb8953e', 'hex')
+    '0000000000001013c28fa079b545fb805f04c496687799b98e35e83cbbb8953e', 'hex'),
+  2016: Buffer.from(
+    '0000000000000424ee6c2a5d6e0da5edfc47a4a10328c1792056ee48303c3e40', 'hex'),
+  10000: Buffer.from(
+    '00000000000001a86811a6f520bf67cefa03207dc84fd315f58153b28694ec51', 'hex'),
+  20000: Buffer.from(
+    '0000000000000162c7ac70a582256f59c189b5c90d8e9861b3f374ed714c58de', 'hex'),
+  30000: Buffer.from(
+    '0000000000000004f790862846b23c3a81585aea0fa79a7d851b409e027bcaa7', 'hex'),
+  40000: Buffer.from(
+    '0000000000000002966206a40b10a575cb46531253b08dae8e1b356cfa277248', 'hex'),
+  50000: Buffer.from(
+    '00000000000000020c7447e7139feeb90549bfc77a7f18d4ff28f327c04f8d6e', 'hex'),
+  56880: Buffer.from(
+    '0000000000000001d4ef9ea6908bb4eb970d556bd07cbd7d06a634e1cd5bbf4e', 'hex'),
+  61043: Buffer.from(
+    '00000000000000015b84385e0307370f8323420eaa27ef6e407f2d3162f1fd05', 'hex')
```

Comment:
Why these specific blocks?

Reply:
✅  Confirmed these hashes match these heights in main chain on node synced without checkpoints:

```
$ for h in 1008 2016 10000 20000 30000 40000 50000 56880 61043 ; do hsd-rpc getblockhash $h; done;
0000000000001013c28fa079b545fb805f04c496687799b98e35e83cbbb8953e
0000000000000424ee6c2a5d6e0da5edfc47a4a10328c1792056ee48303c3e40
00000000000001a86811a6f520bf67cefa03207dc84fd315f58153b28694ec51
0000000000000162c7ac70a582256f59c189b5c90d8e9861b3f374ed714c58de
0000000000000004f790862846b23c3a81585aea0fa79a7d851b409e027bcaa7
0000000000000002966206a40b10a575cb46531253b08dae8e1b356cfa277248
00000000000000020c7447e7139feeb90549bfc77a7f18d4ff28f327c04f8d6e
0000000000000001d4ef9ea6908bb4eb970d556bd07cbd7d06a634e1cd5bbf4e
00000000000000015b84385e0307370f8323420eaa27ef6e407f2d3162f1fd05
```